doc-content/enterprise-only/openshift/operator-createapp-proc.adoc

```diff
@@ -0,0 +1,154 @@
+    - name: server
+      deployments: 2
+      env:
+        - name: MY_VALUE
+          value: "example"
+      # Override default memory limits for all the kie servers in this set
+      resources:
+        limits:
+          memory: 2Gi
+  smartRouter:
+    env:
+      - name: MY_VALUE
+        value: "example"
+----
++
+Replace `<environment_type>` with the type of environment that you want to configure.
++
+** The following snippet configures {KIE_SERVERS}, a {CENTRAL} or {CENTRAL} Monitoring, and a Smart Router using existing secrets for HTTP communication, as required for a production environment. In this example, two servers are created with the `server-a-keystore` secret. (For instructions about creating the secrets, see <<secrets-central-create-proc>>, <<secrets-kie-create-proc>>, and <<secrets-smartrouter-create-proc>>.)
++
+[subs="attributes,verbatim,macros"]
+----
+metadata:
+  name: keystore-config
+spec:
+  environment: <environment_type>
+  objects:
+    console:
+      keystoreSecret: console-keystore
+    servers:
+      - name: server-a
+        deployments: 2
+        keystoreSecret: server-a-keystore
+      - name: server-b
+        keystoreSecret: server-b-keystore
+    smartRouter:
+      keystoreSecret: smartrouter-keystore
+----
++
+Replace `<environment_type>` with the type of environment that you want to configure.
++
+** The following snippet sets the password for the administrator user (`admin`) and the application name (`app2`):
++
+[subs="attributes,verbatim,macros"]
+----
+  commonConfig:
+    adminPassword: password
+    applicationName: app2
+----
++
+** The following snippet sets up LDAP authentication. The parameters correspond to the settings of the LdatExtended Login module of {EAP}. For instructions about using these settings, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/login_module_reference/#ldapextended_login_module[LdapExtended Login Module]. 
++
+[subs="attributes,verbatim,macros"]
+----
+auth:
+  ldap:
+    url: ldaps://myldap.example.com
+    bindDN: uid=admin,ou=users,ou=exmample,ou=com
+    bindCredential: s3cret
+    baseCtxDN: ou=users,ou=example,ou=com
+    baseFilter: (uid={0})
+    searchScope: SUBTREE_SCOPE
+    roleAttributeID: memberOf
+    rolesCtxDN: ou=groups,ou=example,ou=com
+    roleFilter: (memberOf={1})
+    defaultRole: guest
+  roleMapper:
+    rolesProperties: /conf/roleMapper.properties
+    replaceRole: true
+----
++
+If the LDAP server does not define all the roles required for your deployment, you can map LDAP groups to {PRODUCT} roles. To enable LDAP role mapping, set the `rolesProperties` value to the fully qualified pathname of a file that defines role mapping, for example, `/opt/eap/standalone/configuration/rolemapping/rolemapping.properties`. You must provide this file and mount it at this path in all applicable deployment configurations; for instructions, see <<rolemapping-proc>>.
```

Comment:
"configurations. For instruction about LDAP role mapping [or whatever], see .."  (better/cleaner, imo)